### PR TITLE
Added Conor's code

### PR DESCRIPTION
**Pull request by clmorrison**
This adds the code for the survival analysis, the PCA outlier detection, and the very brief kmeans result

### CODE REVIEW

File: src/ConorWork.R [hunk 1]
```diff
@@ -0,0 +1,311 @@
+library(gridExtra)
```
Comment (creisle) [inline src/ConorWork.R:1]: it looks like there are some new R libraries. can you add the code to install them at the top (you can probably copy/paste and work off what @FaezeK has done for the other R scripts)

**Review by creisle — CHANGES_REQUESTED** [review]
A more general thing, can you rename this file to be more descriptive about what it does?
for example something like `survival_analysis.R` would work

File: src/ConorWork.R (hunk 1 above)
Comment (clmorrison) [reply src/ConorWork.R:1]: Sure I can do this tomorrow

**Comment by clmorrison** [comment]
> A more general thing, can you rename this file to be more descriptive about what it does?
> for example something like `survival_analysis.R` would work

I mentioned in slack that my 3 scripts rely on a data formatting script. To avoid needing to save the data file I was thinking it would be easier to just throw everything into a single file.

We can change it to have separate files, though there's a chance it might require debugging on other people's systems. We can cross that bridge when we get there. I'll put these changes through tomorrow

**Pull request closed without merging**